aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,108 @@
+---
+uid: signalr/get-started-signalr-core
+title: Get Started with SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: In this tutorial, you create a SignalR on ASP.NET Core application.
+manager: wpickett
+ms.date: 02/23/2018
+ms.topic: tutorial
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+ms.custom: mvc
+---
+# Tutorial: Get started with SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+This tutorial teaches the basics of building a real-time app using SignalR for ASP.NET Core.
+
+   ![Solution](get-started-signalr-core/_static/signalr-get-started-finished.png)
+
+[View or download the sample](get-started-signalr-core/sample/). [Download instructions](/tutorials/#how-to-download-a-sample)
+
+This tutorial demonstrates the following SignalR development tasks:
+
+> [!div class="checklist"]
+> * Create a SignalR on ASP.NET Core web app.
+> * Create a SignalR hub to push content to clients.
+> * Use the SignalR JavaScript library to send messages and display updates from the hub.
+
+# Prerequisites
+
+Install the following:
+
+* [.NET Core 2.1.0 Preview 1 SDK](https://www.microsoft.com/net/core) or later.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload.
+
+## Create an ASP.NET Core project that hosts SignalR client and server
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**.
+1. Name the project `SignalRChat`
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-dialog.png)
+
+1. Select **Web Application** which creates a project using Razor Pages. Then select **Ok**. Be sure that **ASP.NET Core 2.1** is selected from the framework selector.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
+
+The libraries that contain SignalR server-side code are included in the project template. You must include the client side JavaScript file.
+
+1. Install the JavaScript client library with [npm](https://www.npmjs.com/). Open a command window at the root of your project and run `npm install`. `npm install` creates a *node_modules* folder and sub-folders for the libraries in the same location as where you run the npm command.
+
+  ```console
+   npm install @aspnet/signalr-client --save
```

Comment:
The module ID is now `@aspnet/signalr`, also `--save` is the default, so you can drop it from the invocation.